src/Middleware/ResponseCaching/test/ResponseCachingPolicyProviderTests.cs

```diff
@@ -72,7 +72,7 @@ public void AttemptResponseCaching_AuthorizationHeaders_NotAllowed()
             var sink = new TestSink();
             var context = TestUtils.CreateTestContext(sink);
             context.HttpContext.Request.Method = HttpMethods.Get;
-            context.HttpContext.Request.Headers[HeaderNames.Authorization] = "Basic plaintextUN:plaintextPW";
+            context.HttpContext.Request.Headers.Authorization = "Basic plaintextUN:plaintextPW";
```

Comment:
@benaadams @JunTaoLuo can we use the string `Placeholder` for the pw here? Looks like this introduced a credscan bug (which is surprising since the creds haven't changed).

Reply:
Bonus, so credscan recognises the new Authorization header; whereas it didn't in the old style?

Reply:
Seems that way!